src/xcode/ENA/ENA/Source/Scenes/App/AppNavigationController.swift

```diff
@@ -103,6 +108,25 @@ extension AppNavigationController: UINavigationControllerDelegate {
 			}
 		})
 	}
+
+	/// If the passed in `UIViewController` is a `NavigationBarOpacityDelegate` and contains a `UIScrollView`,
+	/// register an observer for the `contentOffset` property so that the navigation bar's `backgroundAlpha` is set as the controller scrolls.
+	///
+	/// - parameter viewController: The controller to regsiter the observer for
+	private func observeScrollView(of viewController: UIViewController) {
+		guard
+			let opacityDelegate = viewController as? NavigationBarOpacityDelegate,
```

Comment:
this is an interesting way to break the guard statement, but it looks very appealing tbh. Thanks for using this trick, I learnt something new 👍